source/entities/Creature.h

```diff
@@ -423,21 +428,18 @@ class Creature: public MovableGameEntity
     std::deque<CreatureAction>      mActionQueue;
     std::vector<Tile*>              mVisualDebugEntityTiles;
 
-
-    //! \brief Allows to keep track of the entity we are currently attacking
-    Tile*                           mAttackedTile;
-    GameEntity*                     mAttackedObject;
+    //! \brief Contains the actions that have already been tested to avoid trying several times same action
```

Comment:
If i get it right, it's the actions that have already been tested during the turn, right?
If I'm correct, i would add this to the member documentation.